scripts/update_teams.py

```diff
@@ -5,13 +5,15 @@
 #  - python
```

Comment:
oh interesting.  did not really even consider that.  which approach do you think this should take?